torchvision/transforms/functional_tensor.py

```diff
@@ -110,10 +113,13 @@ def adjust_contrast(img, contrast_factor):
     Returns:
         Tensor: Contrast adjusted image.
     """
+    if contrast_factor < 0:
+        raise ValueError('contrast_factor ({}) is not non-negative.'.format(contrast_factor))
+
     if not _is_tensor_a_torch_image(img):
         raise TypeError('tensor is not a torch image.')
 
-    mean = torch.mean(rgb_to_grayscale(img).to(torch.float))
+    mean = rgb_to_grayscale(img).mean()
```

Comment:
mean is currently not supported for `uint8` types. I think this will be soon fixed in PyTorch, but in the meantime, can you revert this change as well?

Reply:
reverted!